### PR TITLE
Also include xray with jQuery3

### PR DESCRIPTION
**Pull request by cerdiogenes**

### CODE REVIEW

File: lib/xray/middleware.rb [hunk 1]
```diff
@@ -100,7 +100,7 @@ def script_matcher(script_name)
       /
         <script[^>]+
         \/#{script_name}
-        (2)?                   # Optional 2 for jQuery2 format
+        (2|3)?                 # Optional 2 for jQuery2 format
```
Comment (mattbrictson) [inline lib/xray/middleware.rb]: Can you also update the comment?

**Comment by cerdiogenes** [comment]
Updated!

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,7 +10,8 @@ lesser changes or bug fixes.
 
 ## [Unreleased][]
 
-* Your contribution here!
```
Comment (mattbrictson) [inline CHANGELOG.md]: Sorry, I should have specified. Can you please put the `* Your contribution here!` back in to guide future contributors? Thanks!

**Comment by cerdiogenes** [comment]
I think everything is fine now!

**Comment by mattbrictson** [comment]
Great, thanks for the contribution! 🙇

**Pull request merged**